api/coordinate_test.go

```diff
@@ -62,3 +65,42 @@ func TestAPI_CoordinateNode(t *testing.T) {
 		// get an error.
 	})
 }
+
+func TestAPI_CoordinateUpdate(t *testing.T) {
+	t.Parallel()
+	c, s := makeClient(t)
+	defer s.Stop()
+
+	node := "foo"
+	_, err := c.Catalog().Register(&CatalogRegistration{
+		Node:    node,
+		Address: "1.1.1.1",
+	}, nil)
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	coord := c.Coordinate()
+	newCoord := coordinate.NewCoordinate(coordinate.DefaultConfig())
+	newCoord.Height = 0.5
+	entry := &CoordinateEntry{
+		Node:  node,
+		Coord: newCoord,
+	}
+	_, err = coord.Update(entry, nil)
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	retryer := &retry.Timer{Timeout: 10 * time.Second, Wait: 1 * time.Second}
```

Comment:
Is the default retry really too slow to be reliable?

Reply:
The coordinate update period (5s) is longer than the default retry. I turned this down to 5s and it seems like enough, so I'll leave it at that.